### DISCUSSION TITLE
Front matter and #webpack_path

### DISCUSSION THREAD

**Discussion by matiaskorhonen** [bridgetownrb/bridgetown#488] [Q&A]
Is it possible to use `webpack_path` with a value from a page's front matter? e.g. `{% webpack_path page.social_image %}`

For example, I'd like to be able to set an image path in the front matter in order to have per-page previews for Twitter et al but at the moment (in v0.21.5) that doesn't seem to be possible with `webpack_path`. I don't know if there's any change to this in v1.0…

I know I can put images in `src/images` and it'll work without `webpack_path`, but that means putting site assets in two different locations and forgoing content hashes for the images.

**Comment by jaredcwhite** [comment]
Hey @matiaskorhonen, sorry I missed seeing this a while back. You could do something like that if you used an ERB or Serbea template in Bridgetown 1.0…wouldn't work in Liquid at the moment though.
